src/distrs/pois.jl

```diff
@@ -19,6 +19,13 @@ poislogpdf(λ::T, x::T) where {T <: Real} = xlogy(x, λ) - λ - loggamma(x + 1)
 
 poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
 
+function poislogcdf(λ::Number, x::Number)
+    return loggamma(floor(x + 1), float(λ)) - logfactorial(floor(x))
+end
+
+poiscdf(λ::Number, x::Number; precision=0) = last(gamma_inc(floor(x + 1), λ, precision))
+poisccdf(λ::Number, x::Number; precision=0) = first(gamma_inc(floor(x + 1), λ, precision))
```

Comment:
Btw, should `precision` be argument rather than kwarg? @devmotion

Reply:
It doesn't matter but IMO it is a bit more natural as a keyword argument since we don't dispatch on it.

Reply:
I don't think we should expose the `precision` argument here. The implementation in SpecialFunctions happens to support it but it's not something that is generally available I don't think it will be in demand here. If it is then I think it will be more natural to get the lower precision version as part of a method for `Float32`.